### PR TITLE
Bartender "Essentials"

### PR DESCRIPTION
**Pull request by musicmanvr**
## About the PR

This PR adds a few drinks that felt strangely absent from the bartender's arsenal while also adding Sol Dry from /tg/.
## Why / Balance

None of these drinks have any special effects, and all exist purely to add more depth to bartender's arsenal.
## Technical details

- 10 New Reagents.
- Sol Dry added to all soda vending machines.
- Energy Drink removed from soda-vend, coconut water added.
## Media

image
From left to right;
Arnold Palmer, Coconut Rum, Cosmopolitan, Painkiller, Pina Colada, Roy Rodgers, Shirley Temple and Blue Hawaiian.

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Nothing crazy, just some new reagents for bartenders to enjoy.
**Changelog**

:cl:
- add: Added Sol Dry, 8 new cocktails and coconut water.

### CODE REVIEW

**Comment by Hanzdegloker** [comment]
But why remove energy drink? D:

**Comment by beck-thompson** [comment]
I don't quite get removing the energy drink but those new drinks look fun!

**Comment by Tayrtahn** [comment]
Nice! If these get merged, I'll make fill levels for them too!

**Comment by OctoRocket** [comment]
I believe that Arnold Palmer is copyrighted, though I'm not sure if using it is against any laws.

**Comment by potato1234x** [comment]
i think sol dry is only meant to be in certain soda machines idk what ones tho

**Comment by Tayrtahn** [comment]
I don't think Arnold Palmer is an issue, tbh. Arizona (iced tea) appears to own the trademark to use Arnold Palmer's signature to market the beverage, but it's been a generic name for that drink for decades. Same deal as the Shirley Temple, actually.

Since we're not using the name to sell a drink, I'm pretty sure we're in the clear, but I'm not a lawyer.

**Comment by musicmanvr** [comment]
> i think sol dry is only meant to be in certain soda machines idk what ones tho

mainly added it to all machines since ginger ale is pretty much a universally drank soda like grape/root beer/lemon lime, plus it gives crew more soda to inevitably empty all over the floor/drink before the machine needs to be restocked

**Comment by musicmanvr** [comment]
> I don't think Arnold Palmer is an issue, tbh. Arizona (iced tea) appears to own the trademark to use Arnold Palmer's signature to market the beverage, but it's been a generic name for that drink for decades. Same deal as the Shirley Temple, actually.
> 
> Since we're not using the name to sell a drink, I'm pretty sure we're in the clear, but I'm not a lawyer.

my main form of reassurance is that tg has been using the name for years now without any issues, that and since we're not making any direct monetary gains off of using the name through a project under open licensing there isn't really any worry in my mind

if it's something that makes people nervous I can change it to half and half

**Comment by mirrorcult** [comment]
names should be fine imo

**Comment by mirrorcult** [comment]
on second thought arnold palmer should be changed but shirley temple is probably fine

**Comment by Plykiya** [comment]
Does the game not support the ñ character in Piña Colada?

File: Resources/Prototypes/Catalog/ReagentDispensers/beverage.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 - type: reagentDispenserInventory
   id: SodaDispenserInventory
   inventory:
+  - DrinkCoconutWaterJug
   - DrinkCoffeeJug
   - DrinkColaBottleFull
   - DrinkCreamCartonXL
   - DrinkDrGibbJug
-  - DrinkEnergyDrinkJug
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Catalog/ReagentDispensers/beverage.yml:8]: Any reason to remove the energy drink?

**Comment by Hanzdegloker** [comment]
aw man, not the energy drink.

**Pull request merged**